cmd/summarizer/main.go

```diff
@@ -75,6 +79,23 @@ func gatherOptions() options {
 	return o
 }
 
+func gcsFixer(ctx context.Context, projectSub string, configPath gcs.Path, gridPrefix, credPath string) (summarizer.Fixer, error) {
+	if projectSub == "" {
+		return nil, nil
+	}
+	parts := strings.SplitN(projectSub, "/", 2)
+	if len(parts) != 2 {
+		return nil, errors.New("malformed project/subscription")
```

Comment:
nit: Are the parameters this fails on noted anywhere? (I didn't see them here or when the error is caught in a parent function)

Reply:
Done